### DISCUSSION TITLE
Align Colums Left or Right

### DISCUSSION THREAD

**Discussion by J0n1k4** [doublecmd/doublecmd#1546] [General]
Hi,

is there a way to align Attributes, Date, Size and Extension to the right, and to let the "Name" column has "relative width". 
This way, when I shrink the DC window, I would still be able to see the mentioned columns, only the "Name" column would shrink accordingly.
As it is now, the "Name" column remains large, and the other columns are hidden (the scroll ribbon is shown at the bottom so I need to scroll right to see other columns).
Thanks
Cheers

**Comment by alexx2000** [comment]
Hi, see [Auto fill columns](https://doublecmd.github.io/doc/en/configuration.html#ConfigViewFull) option and [Custom columns](https://doublecmd.github.io/doc/en/configuration.html#ConfigColumns).

**Comment by J0n1k4** [comment]
Hi Alex,
Auto fill columns > Auto size column: First
did the trick.
Thanks for this tip and for your great work.
